app/controls/actionlabel.cpp

```diff
@@ -0,0 +1,51 @@
+#include "actionlabel.h"
+
+#include <QAction>
+
+ActionLabel::ActionLabel(QWidget *parent) :
+	ActiveLabel(parent),
```

Comment:
You seem to have a mixture of tabs and spaces in the files. Not a big deal, but I would prefer a space only solution. So I would be grateful if you could reformat the affected source files.